Purpose: Nvidia GPU arch 20 21(20) are nolonger supported by CUDA 9.0+
Forward: no need.

diff --git a/cmake/Cuda.cmake b/cmake/Cuda.cmake
index b2b19e8..2fbc5ba 100644
--- a/cmake/Cuda.cmake
+++ b/cmake/Cuda.cmake
@@ -4,7 +4,7 @@ endif()
 
 # Known NVIDIA GPU achitectures Caffe can be compiled for.
 # This list will be used for CUDA_ARCH_NAME = All option
-set(Caffe_known_gpu_archs "20 21(20) 30 35 50 60 61")
+set(Caffe_known_gpu_archs "30 35 50 60 61")
 
 ################################################################################################
 # A function for automatic detection of GPUs installed  (if autodetection is enabled)
